src/components-examples/material/checkbox/checkbox-overview/checkbox-overview-example.html

```diff
@@ -1 +1,25 @@
-<mat-checkbox>Check me!</mat-checkbox>
+<section class="example-section">
+  <mat-checkbox class="example-margin">Check me!</mat-checkbox>
+  <mat-checkbox class="example-margin" [disabled]=true>Disabled</mat-checkbox>
```

Comment:
The `[disabled]` here should have quotes.

Reply:
Done